striketracker/config_test.go

```diff
@@ -0,0 +1,60 @@
+package striketracker
+
+import (
+	"testing"
+)
+
+const (
+	ConfigTestToken = "98fg798hu7sd9hu32kj23kj23"
+	ConfigTestID    = "TestApplication"
+)
+
+func TestDefaults(t *testing.T) {
+
+	config, err := NewConfiguration()
+
+	if err != nil {
+		t.Errorf("Did not expect error")
+	}
+
+	if config.Debug {
+		t.Errorf("Expected Debug to be false")
+	}
+
+	if config.ApplicationID != "" {
+		t.Errorf("Expected App Id to be empty")
+	}
+
+	if config.AuthorizationHeaderToken != "" {
+		t.Errorf("Expected token to be empty")
+	}
+}
+
+func TestSettingOptions(t *testing.T) {
+	config, err := NewConfiguration(WithDebug(true), WithApplicationID(ConfigTestID), WithAuthorizationHeaderToken(ConfigTestToken))
+
+	if err != nil {
+		t.Errorf("Did not expect error")
```

Comment:
"Did not expect error but received %v", err

also for the others you should show what you expected and what you got (even if its obvious).

```
	if config.ApplicationID != ConfigTestID {
		t.Errorf("Expected ApplicationId to be %s but got %s", ConfigTestID, config.ApplicationID)
	}
```

Reply:
yea, I was being lazy, also I'm used test assert functions that do that for me ;)